actors/builtin/power/power_actor.go

```diff
@@ -590,10 +470,44 @@ func (a Actor) OnEpochTickEnd(rt Runtime, _ *adt.EmptyValue) *adt.EmptyValue {
 	return nil
 }
 
+func (a Actor) UpdatePledgeCollateralTally(rt Runtime, pledgeAmount abi.TokenAmount) *adt.EmptyValue {
+	rt.ValidateImmediateCallerType(builtin.StorageMinerActorCodeID)
+	msgValue := rt.Message().ValueReceived()
+	if !msgValue.IsZero() {
+		rt.Abortf(exitcode.ErrIllegalArgument, "fund is not required to update pledge collateral tally")
+	}
```

Comment:
This is silly. We could put a similar check on all methods that don't expect a value, but we don't. Writing a test for the miner would be a great alternative.